components/analytics-processors/org.wso2.carbon.analytics.spark.ui/src/main/java/org/wso2/carbon/analytics/spark/ui/client/AnalyticsExecutionClient.java

```diff
@@ -107,8 +110,12 @@ public void executeScriptContentInBackground(String scriptContent) throws Remote
     }
 
     public AnalyticsProcessorAdminServiceStub.AnalyticsQueryResultDto[] executeScript(String scriptName)
-            throws RemoteException, AnalyticsProcessorAdminServiceAnalyticsProcessorAdminExceptionException {
-        return stub.executeScript(scriptName);
+            throws Exception {
+        try {
+            return stub.executeScript(scriptName);
+        } catch (AnalyticsProcessorAdminServiceAnalyticsProcessorAdminExceptionException e) {
```

Comment:
Why do we use this type of name for an exception class? It is not up to the standard naming convention.

Reply:
This is an exception class in the stub. Since it is already generated, I had to use it for the specific fix. It's resolved in later releases.